### PR TITLE
FIX: issue #4889 (Driver Internal Error: Script Error : status needs …

### PR DESCRIPTION
**Pull request by loziniak**
…a value)

Not tested locally, as it would require a Rebol licence, if I'm not mistaken. So, hope for the best.

### CODE REVIEW

File: red.r [hunk 1]
```diff
@@ -83,7 +83,7 @@ redc: context [
 					
 					IsProcessorFeaturePresent: make routine! [feat [integer!] return: [integer!]] kernel32 "IsProcessorFeaturePresent"
 
-					gui-sys-call: func [cmd [string!] args [string!]][
+					gui-sys-call: func [cmd [string!] args [string!] return: [integer!]][
```
Comment (hiiamboris) [inline red.r:86]: this I think will error out for `return:`

Comment (hiiamboris) [reply red.r:86]: and the other `func`

Comment (loziniak) [reply red.r:86]: Seems probable:
```
% echo "Rebol [] a: func [return: [integer\!]] []" | ./rebol
...
** Script Error: Invalid argument: return
** Where: throw-on-error
** Near: func [return: [integer!]] []
```

Comment (loziniak) [reply red.r]: I deleted `return:`s in `func`s, as they were needed only in `routine`s. Force-pushed the changes.

**Pull request merged**